packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt

```diff
@@ -34,33 +36,45 @@ import org.unimodules.interfaces.permissions.PermissionsResponse
 import org.unimodules.interfaces.permissions.PermissionsResponseListener
 import org.unimodules.interfaces.permissions.PermissionsStatus
 import java.io.IOException
-import java.lang.RuntimeException
 import java.lang.ref.WeakReference
 
 class ImagePickerModule(private val mContext: Context,
-                        val moduleRegistryPropertyDelegate: ModuleRegistryPropertyDelegate = ModuleRegistryPropertyDelegate())
-  : ExportedModule(mContext), ActivityEventListener {
+                        val moduleRegistryPropertyDelegate: ModuleRegistryPropertyDelegate = ModuleRegistryPropertyDelegate(),
+                        private val pickerResultStore: PickerResultsStore = PickerResultsStore(mContext))
+  : ExportedModule(mContext), ActivityEventListener, LifecycleEventListener {
```

Comment:
```suggestion
  val moduleRegistryPropertyDelegate: ModuleRegistryPropertyDelegate = ModuleRegistryPropertyDelegate(),
  private val pickerResultStore: PickerResultsStore = PickerResultsStore(mContext)
) : ExportedModule(mContext), ActivityEventListener, LifecycleEventListener {
```
I would go with this formatting 💅